### PR TITLE
Contact form branch

### PR DESCRIPTION
**Pull request by why-a-en**
In this branch
- We add a header like description to feel free to contact me
- We add a form that takes up three inputs of thier name, email address and the message they wish to leave
- The form is ended with a submit button 
- The whole webpage is finalized with a footer containing social links

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,6 +242,46 @@ <h1>About me</h1>
             </ul>
         </div>
     </section>
+    <!--Contact-form-section-->
+    <section class="contact-form">
+        <form action="https://formspree.io/f/xayzobow" method="post" >
+            <h1 id="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+        <fieldset>
```
Comment (hiromataba) [inline index.html:249]: - [ ] Please update your design to match the original design  the colors, outline for inputs and margin and colors

- Your Design 👇 

image

- Original Design 👇 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @anbehindY    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -242,6 +242,46 @@ <h1>About me</h1>
...
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/twitter1.svg" alt="Twitter logo" /></a>
+                </li>
+                <li>
+                    <a class="logo" href=""><img src="icons/medium1.svg" alt="Medium logo" /></a>
+                </li>
+            </ul>
+    </footer>
 </body>
```
Comment (AbayomiOlaoye) [inline index.html:286]: - [ ] Good job, @anbehindY 👍 however, just like the previous reviewer mentioned, your form is not consistent with the wireframe yet. I have helped to prepare a more explanatory schedule for comparison:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|image|image|

- Kindly notice the following:
  - The `border-bottom` line separating the inputs are too dark than the wireframe is.
  - Your `textarea` has physible border lines which are not present in the requirement.
  -  You seem to be missing the last aesthetic design at the the very bottom of the page.
 
Keep the good work, Champ 💯

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @anbehindY 🥇 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights
- Great response to feedback from previous code reviewers ✅ 
- Great commit messages ✅ 
- No linter error ✅ 
- Nice implementation ✔️ 

 ## Required Changes ♻️
- [x] _Check the comments under the review._

## Optional suggestions
_N/A._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @anbehindY 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline comment.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @anbehindY, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter error. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**